tests/scripts/thread-cert/config.py

```diff
@@ -64,8 +64,9 @@
 DOMAIN_PREFIX_ALTER = 'fd00:7d04:7d04:7d04::/64'
 
 PORT_OFFSET = int(os.getenv('PORT_OFFSET', '0'))
```

Comment:
I wonder why it's named as 'port'. is it due to a reuse of the env variable?

Reply:
not sure the history here, but looks like it's related to the test cases parallel running